Sources/carton/Commands/Local.swift

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 Carton contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArgumentParser
+import TSCBasic
+
+struct Local: ParsableCommand {
+  static var configuration = CommandConfiguration(abstract: "")
```

Comment:
Please add an abstract string for this subcommand:

```suggestion
  static var configuration = CommandConfiguration(
    abstract: """
    Prints SDK version used for the current project or saves it \
    in the `.swift-version` file if a version is passed as an argument.
    """
  )
```